### DISCUSSION TITLE
db1000n stops with Killed status

### DISCUSSION THREAD

**Discussion by daojah** [arriven/db1000n#560] [General]
Hello everyone I have a problem with db1000n after run it's just stop after some time with Killed status. Maybe anyone have similiar issues?

**Comment by arriven** [comment]
The only scenario in which db1000n gets killed that I know of is when it gets OOM killed. It usually only needs 512 mb of RAM to work (or 200 mb if you disable encryption) but there's been reports about OOM even on machines with more resources available (#555)
